redactrus.go

```diff
@@ -56,9 +57,20 @@ func (h *Hook) Fire(e *logrus.Entry) error {
 			}
 
 			// Redact based on value matching in Data fields
+			// Handle fmt.Stringer type
+			if vv, ok := v.(fmt.Stringer); ok {
+				e.Data[k] = re.ReplaceAllString(vv.String(), "$1[REDACTED]$2")
+				continue
+			}
```

Comment:
Can you please add a test that covers this case?

Reply:
Yep, was thinking the same